### PR TITLE
Add guide about `resources`

### PR DESCRIPTION
**Pull request by FabianLars**

### CODE REVIEW

File: docs/guides/building/resources.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+```
+
+## Accessing files in JavaScript
+
+In this example we want to bundle additional i18n json files that look like this:
+
+```json title=de.json
+{
+  "hello": "Guten Tag!",
+  "bye": "Auf Wiedersehen!"
```
Comment (lorenzolewis) [inline docs/guides/building/resources.md:49]: ❤️🇩🇪

Comment (FabianLars) [reply docs/guides/building/resources.md:49]: took me way to long to come up with an example...

**Comment by thegame4craft** [comment]
how is the file & folder structure in `Embedding additional files`?
is it like
```
src/
    ... (frontent-files)
src-tauri/
    lang/
        de.json
    main.rs
```
(shorten)

**Comment by FabianLars** [comment]
yes, but it doesn't matter as long as you change the paths in the scope and resolveResource accordingly.

**Comment by thegame4craft** [comment]
ok because i get errors like
```
resources.ts:16 
        
    Uncaught (in promise) path not allowed on the configured scope: \\?\C:\Users\<USERNAME>\projects\TauriApp2\tauri-app\src-tauri\target\debug\locales\language.json
```

**Comment by FabianLars** [comment]
Ah yes, i copied the wrong scope config... Gonna update that asap, thank you! as a quick fix just change it to `"scope": ["$RESOURCE/*"]`

**Comment by FabianLars** [comment]
Merging due to popular demand. We can fix any spelling mistakes etc in a follow-up PR 🤷

**Pull request merged**